core/templates/dev/head/pages/exploration_player/HintsTooltipService.js

```diff
@@ -17,14 +17,14 @@
  */
 
 oppia.factory('HintsTooltipService', [
-  '$uibModal', '$uibModalStack', '$timeout', 'UrlInterpolationService',
-  function($uibModal, $uibModalStack, $timeout, UrlInterpolationService) {
+  '$timeout',
+  function($timeout) {
     // 10 second wait before closing the modal.
     var CLOSE_TOOLTIP_MSEC = 10000;
-    var WAIT_FOR_TOOLTIP_TO_BE_SHOWN_MSEC = 5000;
+    var WAIT_FOR_TOOLTIP_TO_BE_SHOWN_MSEC = 120000;
     var timeout = null;
     var tooltipIsOpen = false;
-    var tooltipCanBeViewed = true;
+    var tooltipIsEnabled = true;
```

Comment:
Please add comments above these to explain what the difference is between "open" and "enabled".